packages/libs/src/services/schemaValidator/schemas/playlistSchema.json

```diff
@@ -68,6 +88,25 @@
         }
       },
       "title": "PlaylistContents"
+    },
+    "USDCPurchaseConditions": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "price": {
+          "type": "integer",
+          "minimum": 0
+        },
+        "splits": {
+          "type": "object",
+          "patternProperties": {
+            "^[1-9A-HJ-NP-Za-km-z]{32,44}$": {
```

Comment:
Going to assume at first glance that this is a well-known regex for base58 solana addresses?

Reply:
yea looks like it. Again copied from the other test data 😅